### ISSUE TITLE
SVGs have their width and height set rendering them unresponsive

### ISSUE THREAD

**Issue by lasseeee** [laravolt/avatar#140]
TL;DR: Setting the width and height constraints directly on a vector image defeats the purpose of being an svg in responsive webdesign.

The SVGs being created have their height and width set, rendering them unable to be natively responsive in browsers. I tried setting height and width to `null` but to no avail. Would it be a solution to remove the width and height from the generated SVG from `->toSvg()` to make them work as SVGs were intended? If constraints are needed it's never a problem to create a container for the SVG, but removing the constraints from the SVG is impossible. I'd be happy to pull request the removal of the two attributes if this is at all an interesting change.

<img width="534" alt="image" src="https://user-images.githubusercontent.com/1869424/205457261-15c309bc-210a-4a63-adbd-66290ed0ddb1.png">

**Comment by uyab** [comment]
Yeah, I agree width and height attributes should be ignored (not rendered) if the value is null. PR are welcome 👨🏼‍💻
